### PR TITLE
[1LP][RFR] Fix db_migrate fixtures & add fixtures.appliance

### PR DESCRIPTION
**Pull request by jkrocil**
{{pytest: -k test_db_migrate -v}}

### CODE REVIEW

**Comment by jkrocil** [comment]
@lcouzens putting your sprout client to use here ^

File: fixtures/appliance.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    """
+    sprout_client = SproutClient.from_config()
+    apps, request_id = sprout_client.provision_appliances(
+        lease_time=lease_time, preconfigured=preconfigured)
+    app = apps[0]
+    app.stop_evm_service()
+    app.extend_db_partition()
+    app.start_evm_service()
+    yield app
+    sprout_client.destroy_pool(request_id)
```
Comment (mfalesni) [inline fixtures/appliance.py]: **OPTIONAL**: `try/finally` so destroy_pool is in finally? (I hope it works like that in CMs ... ). Can be done in another PR, this is just a suggestion

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by jkrocil** [comment]
56z passed testing, 57z in progress but a few failures is really not a problem as long as the tests run and majority passes.

**Pull request merged**